gatsby-theme-landing-page/src/utils/index.js

```diff
@@ -0,0 +1,41 @@
+import * as sanitizeHTML from "sanitize-html";
```

Comment:
Generally, I'd say you don't need a directory until you have more than one file in it. This could be `src/utils.js`

Reply:
good point. I like the idea of MarkdownText encapsulating this logic anyway, so I'll get rid of this file for now and we can revisit if there's a need